src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -126,10 +126,18 @@ public void handle(Endpoint<?> endpoint) {
                     this.pollTimeOut = Long.parseLong(routingContext.request().getHeader("timeout"));
                 }
 
+                if (routingContext.request().getParam("max_bytes") != null) {
+                    this.maxBytes = Long.parseLong(routingContext.request().getParam("max_bytes"));
+                }
+
                 this.consume(records -> {
                     if (records.succeeded()) {
                         Buffer buffer = (Buffer) messageConverter.toMessages(records.result());
-                        sendConsumerRecordsResponse(routingContext.response(), buffer);
+                        if (buffer.getBytes().length > this.maxBytes) {
+                            routingContext.response().setStatusMessage("Response is too large").setStatusCode(ErrorCodeEnum.UNPROCESSABLE_ENTITY.getValue()).end();
```

Comment:
We should try to put the sending logic in the sendConsumerRecordsResponse refactoring it if needed with more parameters. I noticed that we should do that for other use cases.

Reply:
But that is not matter of this PR, right?

Reply:
Ok, I've done it. It was just a little change. I thought the refactoring would influence more of codebase.